packages/@lwc/synthetic-shadow/src/polyfills/event-target/polyfill.ts

```diff
@@ -19,22 +19,26 @@ import { getEventListenerWrapper } from '../../shared/event-target';
 
 function patchedAddEventListener(
     this: EventTarget,
-    _type: string,
-    _listener: EventListenerOrEventListenerObject,
-    _optionsOrCapture?: boolean | AddEventListenerOptions
+    type: string,
+    listener: EventListenerOrEventListenerObject,
+    optionsOrCapture?: boolean | AddEventListenerOptions
 ) {
     if (isHostElement(this)) {
         // Typescript does not like it when you treat the `arguments` object as an array
         // @ts-ignore type-mismatch
         return addCustomElementEventListener.apply(this, arguments);
     }
-    const args = ArraySlice.call(arguments);
-    if (args.length > 1) {
-        args[1] = getEventListenerWrapper(args[1]);
+    if (arguments.length < 2) {
+        // The browser throws a TypeError for insufficient arguments, so we should too.
+        // https://googlechrome.github.io/samples/event-listeners-mandatory-arguments/
+        throw new TypeError(
```

Comment:
if you call the original listener, then they get the original error, that's what we do in locker everywhere when we determine that the arguments are invalid or incomplete.

Reply:
OK, done.